src/components/StrongPasswordInput/index.js

```diff
@@ -0,0 +1,248 @@
+                    onFocus={onFocus}
+                    onBlur={onBlur}
+                    onClick={onClick}
+                    onKeyDown={onKeyDown}
+                    disabled={disabled}
+                    readOnly={readOnly}
+                    required={required}
+                    maxLength={maxLength}
+                    minLength={minLength}
+                    pattern={pattern}
+                    autoComplete={autoComplete}
+                    aria-labelledby={getInlineTextLabelId()}
+                    aria-describedby={getErrorMessageId()}
+                    isBare={isBare}
+                    isCentered={isCentered}
+                    iconPosition={iconPosition}
+                    icon={icon}
+                    error={error}
+                    status={status}
+                    ref={ref}
+                />
+            </RelativeElement>
+            <StrengthBar passwordState={passwordState} passwordStateLabel={passwordStateLabel} />
+            <RenderIf isTrue={!!bottomHelpText}>
+                <HelpText alignSelf="center" style={{ margin: 0 }}>
+                    {bottomHelpText}
+                </HelpText>
+            </RenderIf>
+            <RenderIf isTrue={!!error}>
+                <ErrorText alignSelf="center" id={getErrorMessageId()}>
+                    {error}
+                </ErrorText>
+            </RenderIf>
+        </StyledContainer>
+    );
+});
+
+StrongPasswordInput.propTypes = {
+    /** Specifies the value of an input element. */
+    value: PropTypes.oneOfType([PropTypes.string, PropTypes.bool]),
+    /** The name of the input. */
+    name: PropTypes.string,
+    /** Text label for the input. */
+    label: PropTypes.oneOfType([PropTypes.string, PropTypes.node]),
+    /** A boolean to hide the input label. */
+    hideLabel: PropTypes.bool,
+    /** Text that is displayed when the field is empty, to prompt the user for a valid entry. */
+    placeholder: PropTypes.string,
+    /** The icon to show if it is passed. It must be a svg icon or a font icon. */
+    icon: PropTypes.node,
+    /** Describes the position of the icon with respect to body. Options include left and right.
+     * This value defaults to left. */
+    iconPosition: PropTypes.oneOf(['left', 'right']),
+    /** The maximum number of characters allowed in the field. */
+    maxLength: PropTypes.number,
+    /** The minimum number of characters allowed in the field. */
+    minLength: PropTypes.number,
+    /** Shows the help message below the input. */
+    bottomHelpText: PropTypes.oneOfType([PropTypes.string, PropTypes.node]),
+    /** Specifies that an input field must be filled out before submitting the form.
+     * This value defaults to false. */
+    required: PropTypes.bool,
+    /** Specifies the regular expression that the input's value is checked against.
+     * This attribute is supported for text, search, url, tel, email, and password types. */
```

Comment:
I think this description line is don't need,  this input is always `type="password"` right?